index.js

```diff
@@ -39,6 +40,19 @@ exports.deploy = async function (options) {
     const account = await near.account(options.accountId);
     let prevState = await account.state();
     let prevCodeHash = prevState.code_hash;
+
+    // Checks if there is an existing contract deployed on this account
+    // (code hash consisting of 32 ones means that the contract code is missing)
+    if (!options.force && prevCodeHash !== '11111111111111111111111111111111') {
+        const answer = await askYesNoQuestion(
+            chalk`{bold.white This account already has a deployed contract [ {bold.blue ${prevCodeHash}} ]. Do you want to proceed? {bold.green (y/n) }}`,
+            false
+        );
+        if (!answer) {
+            process.exit(1);
```

Comment:
We are trying not to terminate the process with `process.exit(x)`. I would suggest restructuring if/else checks.

Reply:
Just for the sake of my curiosity can you elaborate on the reasoning here? I saw this pattern being used in some other places in the codebase so that is why I though it was okay to do so.

Reply:
`near-cli` is used for scripting, so I'm trying not to mess up with error codes.

Reply:
So, now `near deploy ...` -> `question` -> `No` will return 0 (success). And it's the right move since nothing is `failed`.
Not super important since people will not use this interactive mode in scripting, but that is a general rule for me.

Reply:
Makes sense, thanks for the explanation.